modules/API/pages/built-in-endpoints.adoc

```diff
@@ -3048,6 +3052,8 @@ Response::
 
 This endpoint allows you to check the status of a query run in xref:3.2@gsql-ref:querying:query-operations.adoc#_detached_mode_async_option[detached mode].
```

Comment:
Let's take these opportunities to remove references to the 3.2 version specifier. I added these by mistake when I first moved us to Antora and have been removing them when I see them ever since.